### PR TITLE
Add best-browser.lang

### PR DESCRIPTION
**Pull request by amychurchwell**
Adds lang file for the new SEO page /firefox/best-browser.

Demo: https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/best-browser/
Bedrock PR: https://github.com/mozilla/bedrock/pull/7334

### CODE REVIEW

**Comment by amychurchwell** [comment]
Langchecker PR: https://github.com/mozilla-l10n/langchecker/pull/890

File: en-US/firefox/best-browser.lang [hunk 1]
```diff
@@ -0,0 +1,117 @@
+## NOTE: Demo page at https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/best-browser/
+## URL: https://bedrock-demo-achurchwell.oregon-b.moz.works/%LOCALE%/firefox/best-browser/
+
+;Find your best browser for speed, privacy and security.
```
Comment (peiying2) [inline en-US/firefox/best-browser.lang]: Can add a comment that this is an HTML page title - usually translation should be short.

File: en-US/firefox/best-browser.lang [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Find your best browser for speed, privacy and security.
+
+
+;So many browser options, but there’s only one that works best for your needs. The best browser for you should offer both speed and privacy protection.
```
Comment (peiying2) [inline en-US/firefox/best-browser.lang]: Can add a comment that this is an HTML page description.

**Comment by amychurchwell** [comment]
@peiying2 thank you for the great feedback! I updated with the requested changes. The bedrock PR also reflects the copy tweaks. :+1:

File: en-US/firefox/best-browser.lang [hunk 3]
```diff
@@ -0,0 +1,114 @@
+## NOTE: Demo page at https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/best-browser/
+## URL: https://bedrock-demo-achurchwell.oregon-b.moz.works/%LOCALE%/firefox/best-browser/
+
```
Comment (peiying2) [inline en-US/firefox/best-browser.lang]: Double line space between entries.

File: en-US/firefox/best-browser.lang [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+# HTML page title
+;Find your best browser for speed, privacy and security.
+Find your best browser for speed, privacy and security.
+
```
Comment (peiying2) [inline en-US/firefox/best-browser.lang]: Double line space between entries.

File: en-US/firefox/best-browser.lang [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+;How to choose the best browser for you.
+How to choose the best browser for you.
+
+
+;The internet has become as essential as electricity and running water, so choosing the best browser for you is more important than ever. The internet is a second office, a teacher and sometimes a medical advisor, even if your actual doctor would prefer you didn’t look up your symptoms online.
+The internet has become as essential as electricity and running water, so choosing the best browser for you is more important than ever. The internet is a second office, a teacher and sometimes a medical advisor, even if your actual doctor would prefer you didn’t look up your symptoms online.
+
+
+;In the mid-nineties, Netscape, Internet Explorer and AOL dominated the landscape. It was a simpler time when the sweet melody of dial-up internet rang across the land. You learned the meaning of patience waiting for web pages to load. Back then, all that mattered was browser speed
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en-US/firefox/best-browser.lang]: The period is still missing.

File: en-US/firefox/best-browser.lang [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+
+;Last but not least, a safe browser should offer tools to help you keep an eye on your accounts. Think of alerts that go straight to your email if any of your accounts get breached or icons that tell you whether a website is encrypted, (i.e., if it’s a good idea to enter your credit number on a shopping site).
+Last but not least, a safe browser should offer tools to help you keep an eye on your accounts. Think of alerts that go straight to your email if any of your accounts get breached or icons that tell you whether a website is encrypted, (i.e., if it’s a good idea to enter your credit number on a shopping site).
+
+
+;Firefox is offering something new to keep you safe: <a href="%(monitor)s">Firefox Monitor</a>. It’s a free service that will alert you if there are any public hacks on your accounts and let you know if your accounts got hacked in the past. Another neat feature is the Green Lock. It looks like a small green icon at the top left side of the browser window. If you’re on Firefox and see the green lock that means the website is encrypted and secure. If the lock is grey, you might want to think twice about entering any sensitive information.
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en-US/firefox/best-browser.lang]: This is still not fixed.

**Comment by peiying2** [comment]
@amychurchwell, a comment should stay with the string it is providing context for.  When I say `entry`, I meant the strings to be translated. Here is an example you can emulate from: https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/campaign-trailhead.lang.

Also, did you fix the period and the sentence that starts with `If.... lock that...`? I didn't see the changes even though you said to have fixed. I want to make sure we are on the same page or that I missed seeing the changes you made. Thanks.

**Comment by amychurchwell** [comment]
@peiying2 apologies for the confusion, I must have not pushed the latest version.
I rebased this branch with master and it should reflect all the requested changes. Feel free to ping me if there are any remaining issues! 🙂 thanks again.

**Comment by peiying2** [comment]
@amychurchwell if there are no more changes to make, please proceed to add the file to other languages.

File: en-US/firefox/best-browser.lang [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+;One easy way to check is to visit a browser’s content setting page and privacy policy. The privacy webpage should outline if your data is shared and why. It’s why the <a href="%(privacy)s">Firefox privacy notice</a> is easy to read and easy to find.
+One easy way to check is to visit a browser’s content setting page and privacy policy. The privacy webpage should outline if your data is shared and why. It’s why the <a href="%(privacy)s">Firefox privacy notice</a> is easy to read and easy to find.
+
+
+;Choosing the best browser for you is a lot like choosing a home. You want to explore your options, do some research and make a decision based on what’s important to you.
+Choosing the best browser for you is a lot like choosing a home. You want to explore your options, do some research and make a decision based on what’s important to you.
+
+
+;At <a href="%(firefox)s"></a>Firefox, we’ve worked hard to build a browser that is twice as fast as before and gives users more control over their online life.
```
Comment (craigcook) [inline en-US/firefox/best-browser.lang]: The link should wrap around "Firefox", spotted in https://github.com/mozilla/bedrock/pull/7334#discussion_r298378971

Comment (peiying2) [reply en-US/firefox/best-browser.lang]: Good call.

**Comment by peiying2** [comment]
@amychurchwell you have not finished the PR. You need to merge to master and cherry-pick to push to production. If unclear, please ask @craigcook to help.

**Comment by amychurchwell** [comment]
@peiying2 thank you! Unfortunately, I can't merge to master because I don't have write access to this repo.

**Comment by peiying2** [comment]
> @peiying2 thank you! Unfortunately, I can't merge to master because I don't have write access to this repo.

@flodolo can grant you access.

**Comment by flodolo** [comment]
Sent invite

**Comment by amychurchwell** [comment]
@flodolo thank you!

**Comment by peiying2** [comment]
@amychurchwell, did you push the pages to production? This is the last step of the process every time you create a new page or update an existing page. Please do this ASAP.

**Comment by amychurchwell** [comment]
@peiying2 I will finish up now. I paused for lunch. Thank you for being so responsive!

**Pull request merged**